src/main/java/task/Task.java

```diff
@@ -0,0 +1,98 @@
+package task;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a task in the Duke application.
+ */
+public class Task {
+    protected String symbol;
+    protected String description;
+    protected boolean isDone;
+    protected LocalDate by;
+    protected LocalDate start;
+    protected LocalDate end;
```

Comment:
Any reason why these variables were put under Task as opposed to the alternative in the Deadline and Event classes?